src/test/java/indices/Pages.java

```diff
@@ -0,0 +1,27 @@
+package indices;
```

Comment:
why is the package called indices? you are storing there different enums, some of them without indices but with pageNames and such. Update package name to smth like 'entities'

Reply:
done